src/Plugin/Block/SubsitesBannerBlock.php

```diff
@@ -39,7 +39,7 @@ class SubsitesBannerBlock extends SubsitesAbstractBlockBase {
   public function build() {
     $build = [];
 
-    if ($this->getSubsite($this->node)) {
+    if ($this->getSubsite()) {
```

Comment:
This function does not have parameters so this can be removed

Reply:
Agree.